packages/language-service/test/lib/diagnostics.spec.ts

```diff
@@ -77,7 +77,7 @@ describe('diagnostics', () => {
                 end: { line: 3, character: 44 },
             },
             message: `cannot resolve imported symbol "ninja" from stylesheet ".${filePathA}"`,
-            severity: 2,
```

Comment:
?

Reply:
language service uses a number based enum to represent diagnostic severity